contracts/DxMgnPool.sol

```diff
@@ -121,6 +122,18 @@ contract DxMgnPool {
         totalPoolSharesCummulative += totalPoolShares;
     }
 
+    function triggerMGNunlock() public{
+        require(currentState() == State.PoolingEnded, "Pooling period is not yet over.");
```

Comment:
Shouldn't we also claim the remaining funds from the dx auctions here? Otherwise we have no logic to recover the funds from the last auction, before the pooling ends.